packages/benchmark/src/testScenario/testDataLoader.ts

```diff
@@ -0,0 +1,31 @@
+import fs from 'fs';
+import path from 'path';
```

Comment:
```suggestion
import fs from 'node:fs';
import path from 'node:path';
```

Reply:
We should probably have an eslint rule for this if we want to enforce this everywhere (tho this package doesn't yet have eslint configured)